Classes/Emogrifier.php

```diff
@@ -649,7 +661,6 @@ function ($matches) use (&$media) {
     private function createXmlDocument()
     {
         $xmlDocument = new \DOMDocument;
-        $xmlDocument->encoding = self::ENCODING;
```

Comment:
Doesn't the encoding field have any effect? Or could we avoid having the meta tag if we keep using $xmlDocument->encoding?